src/server/page_server.hpp

```diff
@@ -152,6 +153,7 @@ class page_server
 	 * the peer transaction server and the check will no longer be valid
 	 */
 	const transaction_server_type m_server_type;
+	const std::string m_underlying_channel_id;
```

Comment:
I think it's not the "channel id" anymore. How about "m_connection_id"? It would be confusing with `m_conn->get_underlying_channel_id ()` in my opinion.

Reply:
It requires `get_channel_id ()` to be modified along.

Reply:
OK, that can be confused. I will follow your suggestion.